UnitTesting/GTMSenTestCase.h

```diff
@@ -350,6 +350,22 @@
 })
 #endif  // XCTAssertNotEqualCStrings
 
+/*!
+ * @define XCTAssertAsserts(expression, ...)
+ * Generates a failure when ((\a expression) does not assert.
+ * If NS_BLOCK_ASSERTIONS is enabled, this test will always pass.
+ * @param expression An expression.
+ * @param ... An optional supplementary description of the failure. A literal NSString, optionally
+              with string format specifiers. This parameter can be completely omitted.
+ */
+#ifndef NS_BLOCK_ASSERTIONS
+	#define XCTAssertAsserts(expression, ...) \
+		_XCTPrimitiveAssertThrowsSpecificNamed(self, expression, @#expression, NSException, \
+																					 NSInternalInconsistencyException, __VA_ARGS__)
+#else
+	#define XCTAssertAsserts(expression, exception_class, ...)
```

Comment:
This likely won't build, `exception_class` isn't gonna be fined (or will have to be defined in the scope the macro is used.

Reply:
Thank you. Good catch. I had built in release, but didn't realize that we had turned off assertions in release. I turned them on in the iOS and macOS projects to match what Xcode normally would have.